### DISCUSSION TITLE
Show Value for recomended difficulty in lazer client (Parity with site)

### DISCUSSION THREAD

**Discussion by N0Y0U2114** [ppy/osu#30955] [Ideas] [answered]
In the beatmap listing on site, the "recommended difficulty" option shows the star rating that is recommended
![image](https://github.com/user-attachments/assets/55555804-ccda-42d3-9f88-84e5dc5de514)
The lazer client does not show the star rating
![image](https://github.com/user-attachments/assets/cd3552dc-aa69-4341-b64b-5d73bbb13b9f)

**Comment by 12beesinatrenchcoat** [comment, marked as answer]
Has been mentioned before. https://github.com/ppy/osu/discussions/30394
> peppy: Definitely something we should have. Implementing is a bit complicated due to how things are currently structured.
